src/auth/dto/login.dto.ts

```diff
@@ -0,0 +1,9 @@
+import { IsString } from 'class-validator';
+
+export class LoginDto {
+  @IsString()
+  username: string;
+
+  @IsString()
+  password: string;
+}
```

Comment:
In my opinion you should use validation for the fields

Reply:
thx
updated it in the next task
https://github.com/anticher/nodejs-st-22-rest-api/commit/ba19e0b5279934b6431673723f682a7c06991510